### ISSUE TITLE
What is the max space available on the NES Mini?

### ISSUE THREAD

**Issue by DecanFrost** [ClusterM/hakchi2#312]
What is the max space we can use for games ?

Yesterday I was at 1016 games ( using 334.1mb of  334.9mb total )

Today I went to swap out a game and now it tells me I have  334.1mb used out of 352.4mb total

So I added some more games , I'm now at 1028 games. Using 347.1mb out of 352.4mb

How is it possible that I keep gaining free space ?

Yes the all synced and work

http://imgur.com/a/mnr4q

**Comment by ClusterM** [comment]
Writable partition is 384 megabytes in size. It contains save-states, settings and all data installed using hakchi. So max free space varies and depends on many factors.

**Comment by DecanFrost** [comment]
On the 7 systems I have tested so far . I have factory reset and made sure no save States are in them. And loaded the exact same set of games and adding and 3 of 7 let me put 349.2mb. 2 let me put 3.51.9mb and 3 others let me put 334.2mb. It would give me an error at the end so I kept removing a game at a time till it completed with out errors  it's very odd that all those systems have the same addons and no save States  and give different results still
